modules/services/mail/offlineimap.nix

```diff
@@ -51,10 +51,9 @@ in {
     environment.etc."offlineimaprc".text = cfg.extraConfig;
     launchd.user.agents.offlineimap = {
       path                            = [ cfg.package ];
-      command                         = "offlineimap";
+      command                         = "${cfg.package}/bin/offlineimap -c /etc/offlineimaprc" + optionalString (cfg.runQuick) "-q";
```

Comment:
I think this needs to be `" -q"` otherwise the flag gets appended to the config file